eea/facetednavigation/widgets/autocomplete/widget.py

```diff
@@ -39,7 +44,20 @@
             i18n_domain="eea"
         )
     ),
-    BooleanField('onlyallelements',
+    StringField(
```

Comment:
As you made this configurable I think an upgrade step is required. See my comment on **eea/facetednavigation/widgets/autocomplete/configure.zcml**